alwaysontop/helpers.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-file
+const isValidWindow = win => win && !win.closed;
+
+const openWindow = async (...args) => {
+  let tries = 5;
+  let win;
+
+  while (!isValidWindow(win) && tries > 0) {
+    tries -= 1;
+
+    win = window.open(...args);
+
+    // eslint-disable-next-line
+    await waitEventRounds(2);
```

Comment:
Why do we need the retry logic? Have you noticed failures to open window?

Reply:
yes, electron will open a window sometimes, but it will be in limbo and there will be an IPC error so the loop circumvents this

Reply:
And why do we need the timeout?